code-list-scripts/cypher/faster-indicators-estimate.cypher

```diff
@@ -0,0 +1,7 @@
+CREATE CONSTRAINT ON (n:`_code_faster-indicators-estimate`) ASSERT n.code IS UNIQUE;
+CREATE CONSTRAINT ON (n:`_code_list_faster-indicators-estimate`) ASSERT n.code IS UNIQUE;
+CREATE (node:`_code_list`:`_code_list_faster-indicators-estimate` { label:"estimate", edition:"one-off" });
+MERGE (node:`_code`:`_code_faster-indicators-estimate` { value:"first-estimate" });
+MATCH (parent:`_code_list`:`_code_list_faster-indicators-estimate`),                (node:`_code`:`_code_faster-indicators-estimate` { value:"first-estimate" })                 MERGE (node)-[:usedBy { label:"First Estimate"}]->(parent);
```

Comment:
Is there any more meaningful label we can provide than first and second? is it like an initial and revised estimate? As a user I would have no idea what these meant to sensibly choose between them (though admittedly, we could clarify that in the dimension description - just checking theres nothing we can do at the code level)